integration-tests/entry-point.sh

```diff
@@ -60,6 +57,25 @@ chmod 640 ${CONFIG_PATH}
 # start ssh
 service ssh start || true
 
+# Ensure lockfile exists
+mkdir -p /run/lock > dev/null
+
+# Spoof lockfile
+echo "${THING_NAME}" > /run/lock/devicecl.lock
+bash -c 'exec -a aws-iot-device-client sleep 1000' &
+PID=$!
+echo ${PID} >> /run/lock/devicecl.lock
+
+# Start and stop Device Client
+./aws-iot-device-client
+retVal=$?
+if [ $retVal -ne 1 ]; then
+    exit 1
```

Comment:
I would add more comments walking through this... This is actually an integration test but it doesn't say that.

Lets explicitly call this out as as a test, explain each step. 

And can you add some stdout for the failure case? If our integration tests just stop running because of a lock file issue we want to know why

Reply:
Modified the test to no longer check for the return code of device client (in the case where device client keeps running and does not exit) but instead checks for the return code of pkill after we kill the dummy process first. This value will be 1 if there is no such process named 'aws-iot-device-client'